appservice/src/deploy/getDeployFsPath.ts

```diff
@@ -81,3 +84,8 @@ async function appendDeploySubpathSetting(targetPath: string, extensionPrefix: s
 
     return targetPath;
 }
+
+export type deployPaths = {
+    deployFsPath: string,
+    deployFsSubpath: string
```

Comment:
Based on your names, I would expect deployFsSubpath to be `bin/release/netcoreapp2.0`, but I believe it's the whole path, right? Can we change the names to be more accurate? How about something like this:

originalDeployFsPath
effectiveDeployFsPath

Either way, you should add comments saying what each path is